sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationEventProcessor/EventResult/RemoveParticipantEventResult.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Communication.CallAutomation
+{
+    /// <summary><see cref="RemoveParticipantEventResult"/> is returned from WaitForEvent of <see cref="RemoveParticipantResult"/>.</summary>
+    public class RemoveParticipantEventResult
+    {
+        /// <summary>
+        /// Indicates whether the returned event is considered successful or not.
+        /// </summary>
+        public bool IsSuccessEvent { get; internal set; }
+
+        /// <summary>
+        /// <see cref="RemoveParticipantSucceeded"/> event will be returned when the participant joined the call successfully.
```

Comment:
Summary to say Removed participant successfully

Reply:
fixed